algorithms/Frequent_Pattern_Mining/fpm/ExprFunctions.hpp

```diff
@@ -0,0 +1,190 @@
+/*Copyright (c) 2015-2016, TigerGraph Inc.
```

Comment:
Pay attention to the copyright year. You are making a change in 2021, so you should update the year.

Reply:
updated